django3_codemods/config_tools/travis_modernizer.py

```diff
@@ -37,44 +44,56 @@ def remove_django_envs(self):
             self.travis_dict = yaml.load(file, Loader=yaml.FullLoader)
             if 'matrix' in self.travis_dict.keys():
                 env_list = self.travis_dict['matrix']['include']
-                updated_list = [
-                    env for env in env_list if not self.DJANGO_PATTERN.search(env['env'])
-                ]
-                self.travis_dict['matrix']['include'] = updated_list
+                if 'env' in env_list:
+                    updated_list = [
+                        env for env in env_list if not self.DJANGO_PATTERN.search(env['env'])
+                    ]
+                    self.travis_dict['matrix']['include'] = updated_list
             if 'env' in self.travis_dict.keys():
                 env_list = self.travis_dict['env']
-                updated_list = [
-                    env for env in env_list if not self.DJANGO_PATTERN.search(env)
-                ]
+                if isinstance(env_list, list):
+                    updated_list = [
+                        env for env in env_list if not self.DJANGO_PATTERN.search(env)
+                    ]
+                else:
+                    updated_list = [
+                        {env: env_list[env]} for env in env_list if not self.DJANGO_PATTERN.search(env)
+                    ]
                 self.travis_dict['env'] = updated_list
             return self.travis_dict
 
     def update_python_version(self):
         """
         Add py38, remove py36 if present.
         """
+        versions_to_remove = [3.5, '3.5', 3.6, '3.6', 3.7, '3.7']
```

Comment:
same here as I mentioned for the tests

Reply:
Same reason as above. Also, in near future, we'll probably need to switch from Travis to Github Actions so we shouldn't spend any more time on these scripts.